config/widgets/settings/generalOptionsTab.json

```diff
@@ -70,6 +70,35 @@
 				}
 			]
 		},
+		{
+			"type" : "verticalLayout",
+			"customType" : "checkboxFake",
+			"position" : {"x": 10, "y": 83},
+			"items" : [
+				{
+					"created" : "desktop"
+				},
+				{},
+				{
+					"created" : "desktop"
+				},
+				{
+					"created" : "desktop"
+				},
+				{},
+				{},
+				{
+					"name": "spellbookAnimationCheckboxPlaceholder"
+				},
+				{
+					"created" : "touchscreen"
+				},
+				{
+					"created" : "mobile"
+				},
+				{}
+			]
+		},
```

Comment:
What's this change about?

Reply:
change from laserlicht - checkbox becomes fake when large spellbook is toggled on

Reply:
Ah, so we have a layer of fake checkboxes below real ones?

Reply:
Exactly. Simplest solution until we have "threestate" checkboxes.